WebUI/src/components/SettingsModel.vue

```diff
@@ -0,0 +1,354 @@
+              languages.COM_DOWNLOAD }}</button>
+        </div>
+        <div class="flex items-center gap-4">
+                          <span class="text-gray-300 flex-auto">{{ languages.SETTINGS_MODEL_IMAGE_RESOLUTION_HD }}:
+                              lcm-lora-sdxl</span>
+          <span class="flex-none text-right">375.61 MB</span>
+          <button class="text-yellow-500 flex-none"
+                  @click="downloadModel('latent-consistency/lcm-lora-sdxl', Const.MODEL_TYPE_LORA)">{{
+              languages.COM_DOWNLOAD }}</button>
+        </div>
+      </div>
+      <div class="flex flex-col gap-3">
+        <p>{{ languages.DOWNLOADER_FOR_ANSWER_GENERATE }}</p>
+        <div class="flex items-center gap-4">
+          <span class="text-gray-300 flex-auto">microsoft/Phi-3-mini-4k-instruct</span>
+          <span class="flex-none text-right">7.11 GB</span>
+          <button class="text-yellow-500 flex-none"
+                  @click="downloadModel('microsoft/Phi-3-mini-4k-instruct', Const.MODEL_TYPE_LLM)">{{
+              languages.COM_DOWNLOAD }}</button>
+        </div>
+      </div>
+      <div class="flex flex-col gap-3">
+        <p>{{ languages.DOWNLOADER_FOR_RAG_QUERY }}</p>
+        <div class="flex items-center gap-4">
+                          <span class="text-gray-300 flex-auto">{{ languages.SETTINGS_MODEL_IMAGE_RESOLUTION_STRANDARD }}:
+                              bge-large-en-v1.5</span>
+          <span class="flex-none text-right">1.25 GB</span>
+          <button class="text-yellow-500 flex-none"
+                  @click="downloadModel('BAAI/bge-large-en-v1.5', Const.MODEL_TYPE_EMBEDDING)">{{
+              languages.COM_DOWNLOAD }}</button>
+        </div>
+        <div class="flex items-center gap-4">
+                          <span class="text-gray-300 flex-auto">{{ languages.SETTINGS_MODEL_IMAGE_RESOLUTION_HD }}:
+                              bge-large-zh-v1.5</span>
+          <span class="flex-none text-right">1.21 GB</span>
+          <button class="text-yellow-500 flex-none"
+                  @click="downloadModel('BAAI/bge-large-zh-v1.5', Const.MODEL_TYPE_EMBEDDING)">{{
+              languages.COM_DOWNLOAD }}</button>
+        </div>
+      </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { useI18N } from "@/assets/js/store/i18n";
+import { useModels } from "@/assets/js/store/models";
+import DropSelector from "../components/DropSelector.vue";
+import FolderSelector from "../components/FolderSelector.vue";
+import { Input } from '@/components/ui/input'
+import { toast } from "@/assets/js/toast";
+import { Const } from "@/assets/js/const";
+import {useGlobalSetup} from "@/assets/js/store/globalSetup.ts";
+const i18n = useI18N();
+const models = useModels();
+const globalSetup = useGlobalSetup();
+
+const presetModel = reactive<StringKV>(Object.assign({}, toRaw(globalSetup.presetModel)));
+const presetModelChange = ref(false);
```

Comment:
this variable name alones raises confusion about the concept of changing a presetModel.
I however like that you operated the complete logic out of appSettings, leaving AppSettings much cleaner.
It probably would be beneficial, if we operated out the "modelSettingsState" into a dedicated class outside of global state, but given that we have so far questions about how this should be used (and the future of it) we should definetly not do it right now